From aa0337f7856feefab804c8a7bba4eaca205d3bab Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 20 Feb 2014 09:48:37 +0000
Subject: [PATCH 1/2] lua: Do not link Lua module to the Lua core library

Modules should not link to the Lua core library.

See http://lua-users.org/wiki/BuildingModules under
"Do Not Link Modules to the Lua Core Libraries"

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 src/lua-lxc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lua-lxc/Makefile.am b/src/lua-lxc/Makefile.am
index 7bbaf61..540238f 100644
--- a/src/lua-lxc/Makefile.am
+++ b/src/lua-lxc/Makefile.am
@@ -18,7 +18,7 @@ core_so_LDFLAGS = \
 	-L$(top_srcdir)/src/lxc \
 	-Wl,-soname,core.so.$(firstword $(subst ., ,$(VERSION)))
 
-core_so_LDADD = -llxc $(LUA_LIBS)
+core_so_LDADD = -llxc
 
 lxc.lua:
 
-- 
1.8.5.4

